portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicPlaylist.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.types.common.models.music;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * POJO for https://schema.org/MusicPlaylist
+ */
+public class MusicPlaylist {
+
+  private final String id;
+  private final String title;
+  private final String description;
+
+  /**
+   * The same as https://schema.org/dateCreated but of uses java.time.Instant for arbitrary granularity usage
+   */
+  private final Instant timeCreated;
+
+  /**
+   * The same as https://schema.org/dateModified but of uses java.time.Instant for arbitrary granularity usage
+   */
+  private final Instant timeUpdated;
+
+  @JsonCreator
+  public MusicPlaylist(
+      @JsonProperty("id") String id,
+      @JsonProperty("title") String title,
+      @JsonProperty("description") String description,
+      @JsonProperty("timeCreated") Instant timeCreated,
+      @JsonProperty("timeUpdated") Instant timeUpdated) {
+    if (isNullOrEmpty(id)) {
+      throw new IllegalArgumentException("id must be for MusicPlaylist");
```

Comment:
```suggestion
      throw new IllegalArgumentException("non-empty id required for MusicPlaylist");
```

Reply:
Added in commit https://github.com/google/data-transfer-project/pull/1131/commits/96bcb7bd984c4c9cc45dc85616d3b0bbe0baf1a4.